src/test/java/seedu/address/model/transaction/DateTest.java

```diff
@@ -39,9 +39,11 @@ public void isValidDate() {
         assertFalse(Date.isValidDate("39/10/2000")); // invalid day
         assertFalse(Date.isValidDate("-11/11/2000")); // negative integers
         assertFalse(Date.isValidDate("11.223/09/2000.0")); // non-integer values
+        assertFalse(Date.isValidDate("31/02/2000")); // Invalid day in february
 
```

Comment:
Great coverage! Maybe adding another test case where the input is the 31st in a month where there are only 30 days would increase coverage. E.g 31/11/2022 Cheers !